pkg/operators/dynamodb/operator.go

```diff
@@ -65,7 +65,13 @@ func QueueUpdater(config config.Config, msg *queuemanager.MessageBody) error {
 		namespace = msg.Namespace
 	} else {
 		clientSet, _ := awsclient.NewForConfig(config.RESTConfig)
-		resources, err := clientSet.DynamoDBs("").List(metav1.ListOptions{})
+
```

Comment:
Actually @pauldthomson noticed something. We need to make sure these bits are included in the code generation so that this is not overwritten the next time that is run. https://github.com/awslabs/aws-service-operator/blob/master/code-generation/pkg/codegen/assets/operator.go.templ#L80